kuma/settings/common.py

```diff
@@ -994,6 +994,12 @@ def pipeline_one_scss(slug, **kwargs):
 
 
 PIPELINE_JS = {
+    'dnt-helper': {
+        'source_filenames': (
+            'js/libs/mozilla.dnthelper.js',
```

Comment:
I can't belive we already had it and weren't using it. *face, palm*

Reply:
It was added with Traffic Cop earlier this year.

Reply:
Okay, I feel less bad now.